cypress/support/login.ts

```diff
@@ -91,7 +94,12 @@ Cypress.Commands.add(
 // eslint-disable-next-line max-len
 Cypress.Commands.add(
     "initTestUser",
-    (synapse: SynapseInstance, displayName: string, prelaunchFn?: () => void): Chainable<UserCredentials> => {
+    (
+        synapse: SynapseInstance,
+        displayName: string,
+        prelaunchFn?: () => void,
+        userIdPrefix = "user_",
```

Comment:
Maybe it should default to `displayName.toLowerCase()`

Reply:
I started out with that, but then a bunch of tests use spaces and special characters and it got hard and I couldn't be bothered.